test/interop/C/multilocale/globals/use_reliesOnGlobal.skipif

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+import os
+
+print('memory' not in  os.getenv('CHPL_SANITIZE_EXE', ''))
```

Comment:
I cannot remember, do we set this environment if the `CHPL_SANITIZE` env was set? If not, these tests won't be skipped for `CHPL_SANITIZE`?

Reply:
I looked a bit and there are some other skipifs like this. And we do set `CHPL_SANITIZE` in our nightly scripts. So this should work... somehow.

Reply:
`CHPL_SANITIZE_EXE` is inferred if `CHPL_SANITIZE` is set and the chplenv is injected into skipifs (by testEnv). e.g.:

```
$ ./util/chplenv/chpl_sanitizers.py --exe
none
$ CHPL_SANITIZE=memory ./util/chplenv/chpl_sanitizers.py --exe
memory

$ ./util/test/testEnv test/interop/C/multilocale/globals/use_reliesOnGlobal.skipif
True
$ CHPL_SANITIZE=memory ./util/test/testEnv test/interop/C/multilocale/globals/use_reliesOnGlobal.skipif
False
```

Reply:
Ah OK.